IdokladSdk.UnitTests/Tests/Validation/Detailed/ModelValidatorTests.CannotEqualIfAttribute.cs

```diff
@@ -0,0 +1,72 @@
+﻿using IdokladSdk.Enums;
+using IdokladSdk.UnitTests.Tests.Validation.Detailed.Model;
+using IdokladSdk.Validation.Attributes;
+using NUnit.Framework;
+
+namespace IdokladSdk.UnitTests.Tests.Validation.Detailed
+{
+    public partial class ModelValidatorTests
+    {
+        [TestCase(0)]
+        [TestCase(0.0)]
+        public void ModelWithCannotEqualIfAttribute_InValidData_ReturnsInValidModel(decimal propertyValue)
```

Comment:
Grammar nazi poznámka. Invalid je jedno slovo, takže bych nepsal InValid s velkým V. To bys pak se stejnou logikou měl psát třeba Cannot jako CanNot

Reply:
Opraveno